src/locutus/model/terminology.py

```diff
@@ -68,17 +68,14 @@ def build_code(self, data, **kwargs):
             # pdb.set_trace()
             return Coding(**data)
 
-    def to_dict(self, valid=None):
+    def to_dict(self):
         obj = {"code": self.code, "display": self.display}
 
         if self.description != "":
             obj["description"] = self.description
 
         if self.system is not None:
             obj["system"] = self.system
-                    
-        if valid:
-            obj["valid"] = valid

```

Comment:
Removing 'valid' from functions used for Coding(terminology). CodingMapping will extend Coding with 'valid' and 'mapping_relationship'